orsopy/fileio/reduction.py

```diff
@@ -0,0 +1,45 @@
+"""
+The reduction elements for the ORSO header
+"""
+
+# author: Andrew R. McCluskey (arm61)
+
+from typing import Optional, List
+from dataclasses import field, dataclass
+import datetime
+from dataclasses_json import dataclass_json
+from .base import Header, Person
+
+
+@dataclass_json
```

Comment:
Do you use any of the functionality provided by this? E.g. it already provides the .to_dict method. If we do this our selves anyway, including this decorator might have undesired side-effects.

Reply:
I haven't tried reading files in, but my logic was that the `from_dict` method from `@dataclass_json` would be useful in file reading in the future.

Reply:
@bmaranville had the suggestion of using `@dataclass_json` so perhaps he can elucidate more.

Reply:
I included dataclass_json just as a proof-of-concept for reading headers back into the python classes - I agree that it probably doesn't belong in our standard library.

Reply:
Fair enough, I thought you were super keen on it but I guess I misread the room.

Reply:
Oh, I do like it very much!  I just though we were trying to avoid third-party libraries in this code.